src/components/EditableField.jsx

```diff
@@ -115,12 +126,21 @@ const EditableField = (props) => {
 
     return (
       <>
-        {renderBold ? <strong>{editedValue}</strong> : <span>{editedValue}</span>}
+        {renderBold ? <strong>{value}</strong> : <span>{value}</span>}
```

Comment:
This was one of the fixes for the problem: we were displaying the internal value `EditableField` uses for keeping track of user actions (`editedValue`) and of course if the request to modify this value fails that isn't really going to cause any re render because nothing changed (`value` is still the same) so `editedValue` wasn't updated.

As a consequence we were left with the latest value the user had put, which was the new changed label that had failed to be updated in the api.

With this change we are going to make sure that the value we are showing is the `value`, (we still need `editedValue` for internal onChange tracking of some edge use cases like when we have external button like save in the cellSets tool creation of new cellSets).